examples/model_simulated/vrops_thermistor_model_simulated.py

```diff
@@ -0,0 +1,158 @@
+thermistor_model = ThermistorModelSimulated(ureg=ureg)
+thermistor_model.run()
+
+#---------------------------------------------------------------------------
+# The following functions operate on physical variables represented in
+# pint objects, and returns a pint object, too.
+# Decorators provided by the pint library are used to check the dimensions of
+# arguments passed to the functions.
+
+@ureg.check(ureg.volt, ureg.ohm, ureg.volt)
+def get_rx(u, r0, ux):
+    rx = r0 * ux / (u - ux)
+    return rx
+
+@ureg.check(1 / ureg.kelvin, 1 / ureg.kelvin, 1 / ureg.kelvin, ureg.ohm)
+def get_temperature(c1, c2, c3, rx):
+    temper = 1 / ( \
+            c1 + \
+            c2 * math.log(rx / ureg.ohm) + \
+            c3 * math.log(rx / ureg.ohm) ** 3
+        )
+
+    #-----------------------------------------------------------------------
+    # Here commented is a counter example, showing how a dimension mismatch
+    # can be prevented using pint.
+    # Since in the correct one above, the unit of temper is 
+    # already Kelvin, if we multiply it by ureg.kelvin, the unit of the
+    # returned values will become ureg.kelvin ** 2, which will consequently 
+    # throw an exception in succeeding method calls.
+
+
+    # temper = 1 / ( \
+    #         c1 + \
+    #         c2 * math.log(rx / ureg.ohm) + \
+    #         c3 * math.log(rx / ureg.ohm) ** 3
+    #     ) * ureg.kelvin
+
+    return temper
+
+#---------------------------------------------------------------------------
+# This is a sampling method, which queries the physical model, and calls the
+# physical functions to calculate a desired variable.
+# In this specific case, it gets the coefficients, voltages, and reference
+# resistance from the thermistor simulator, and calls the methods defined
+# above to get the temperature.
+
+def get_thermistor_temperature():
+    temper = get_temperature(
+            thermistor_model.get_c1(),
+            thermistor_model.get_c2(),
+            thermistor_model.get_c3(),
+            get_rx(
+                    thermistor_model.get_u(),
+                    thermistor_model.get_r0(),
+                    thermistor_model.get_ux()
+                )
+        ).to(ureg.degC)
+    return temper.magnitude
+
+#---------------------------------------------------------------------------
+# In this example, we demonstrate how simulated data can be directed to vROps,
+# VMware's data center component using Liota.
+
+if __name__ == '__main__':
+
+    # create a data center object, vROps in this case, using websocket as a transport layer
+    # this object encapsulates the formats and protocols neccessary for the agent to interact with the dcc
+    # UID/PASS login for now.
+    vrops = Vrops(config['vROpsUID'], config['vROpsPass'], WebSocket(url=config['WebSocketUrl']))
+
+    # create a gateway object encapsulating the particulars of a gateway/board
+    # argument is the name of this gateway
+    gateway = Dk300(config['Gateway1Name'])
```

Comment:
Are you able to successfully run this program? We have changed sampleProp.conf to have a [SECTION] and are reading it using ConfigParser in other sample programs.